### ISSUE TITLE
Class instance of bitarray lead to Python interpreter crashes when executing flip operations.

### ISSUE THREAD

**Issue by xiaxinmeng** [ilanschnell/bitarray#116]
Class instance of bitarray lead to Python interpreter crashes when executing flip operations.
------------------------------------------
>>import bitarray
>>a = bitarray.bitarray(10)
>>b = 1
>>b^a
segmentation fault:11
------------------------------------------
System: Mac OS, Python 3.7, 3.8, 3.9

**Comment by xiaxinmeng** [comment]
Thank you for your kind explanations.  Please let me know if you figure out why it crashes Python 3 interpreter.

**Comment by ilanschnell** [comment]
Yes, I figured it out!  And fixed it on master already, see commit 2eff1b42cb4b5b175d42472bc51960119e7a6e5
This crash wasn't covered by any tests, so thanks again for reporting.  I'm working on improving the test suite and making this fix part of the 1.9.0 release.

So in the next release you will get:
```
>>> from bitarray import bitarray
>>> a = bitarray(10)
>>> 1 ^ a
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for ^: 'int' and 'bitarray.bitarray'
```
Instead of a nasty crash!

**Comment by ilanschnell** [comment]
As this is extremely important fix, I've just released 1.8.2.  Please me know if you have any problems, and thanks again.
